pages/History.qml

```diff
@@ -769,14 +760,44 @@ Rectangle {
 
                                 MoneroComponents.TextPlain {
                                     font.family: MoneroComponents.Style.fontRegular.name
-                                    font.pixelSize: 14
-                                    text: blockheight > 0 ? blockheight : qsTr('Pending') + translationManager.emptyString;
+                                    font.pixelSize: 15
+                                    text: {
+                                        if (isout) {
+                                            if (address !== "") {
```

Comment:
minor but just `if (address)` seems a bit easier to read

inside the if you could also use the ternary operator

Reply:
i agree